### PR TITLE
feat: Fix Navigation tree behavior issues - EXO-64720 - Meeds-io/MIPs#51

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/NodeListTableItem.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
           <v-row
             slot-scope="{ hover }"
             class="d-flex pt-2 px-0 text-truncate v-list-item v-list-item--dense d-flex flex-nowrap"
-            :class="`${highlightNode ? 'light-grey-background ' : ' '}`">
+            :class="`${highlightNode ? 'light-grey-background ' : ' ' } ${expanded ? ' ' : ' ms-4 me-1 '} `">
```
Comment (azayati) [inline layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/NodeListTableItem.vue]: computed

File: layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/NodeListTableItem.vue [hunk 2]
```diff
@@ -22,7 +22,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
           <v-row
```
Comment (azayati) [inline layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/NodeListTableItem.vue]: IMO we should keep the name NodeItem.vue

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**